docs/swagger/models/FederateInfo.yaml

```diff
@@ -0,0 +1,60 @@
+title: FederateInfo
```

Comment:
This is going to be a beast, right? So many options to support or do we have a better way?

Reply:
All the specific options that are not flags are listed already.   I will probably expand this later to include the flags.
This is eventually meant to document the json configuration files but not fully there yet.